thirdparty/enet/enet/godot.h

```diff
@@ -72,9 +72,9 @@ typedef void ENetSocketSet;
 
 typedef struct _ENetAddress
 {
-   uint8_t host[16];
-   uint16_t port;
-   uint8_t wildcard;
+   uint8_t host[16] = {};
+   uint16_t port = 0;
+   uint8_t wildcard = 0;
```

Comment:
![image](https://github.com/godotengine/godot/assets/4701338/f61558e0-dc48-4236-b6be-25f664f22e16)

CC @Faless

Reply:
This is C code so my fix was invalid, would need a different approach. Removed from this PR for now.